databricks_cli/dbfs/api.py

```diff
@@ -123,8 +134,48 @@ def get_file(self, dbfs_path, dst_path, overwrite, headers=None):
                 offset += bytes_read
                 local_file.write(b64decode(data))
 
+    def get_num_files_deleted(self, partial_delete_error):
```

Comment:
Could we add a unit test for this function?

Reply:
Done.